### PR TITLE
Create a Region component to list a table

### PR DESCRIPTION
**Pull request by priyesh777**
### Involves

- Add Region Table that shows the list of Regions along with it's statistical data
- Add Monitoring Expert Modal
- Add Regional Coordinator Modal

### CODE REVIEW

**Review by tnagorra — APPROVED** [review]

File: src/views/Admin/UserRoles/RolesAction/index.tsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+    } = props;
+
+    const handleToggleRoleStatus = useCallback(
+        () => {
+            if (onToggleRoleStatus) {
+                if (roleStatus && roleStatus === 'ADMIN') {
+                    onToggleRoleStatus(id, false);
+                } else {
+                    onToggleRoleStatus(id, true);
+                }
```
Comment (tnagorra) [inline src/views/Admin/UserRoles/RolesAction/index.tsx:41]:
```suggestion
                onToggleRoleStatus(id, roleStatus === 'ADMIN');
```

File: src/views/Admin/UserRoles/RolesAction/index.tsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            }
+        },
+        [onToggleRoleStatus, id, roleStatus],
+    );
+
+    return (
+        <Actions className={className}>
+            {children}
```
Comment (tnagorra) [inline src/views/Admin/UserRoles/RolesAction/index.tsx:49]: We don't need to support children

File: src/views/Admin/UserRoles/RolesAction/index.tsx [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+            {onToggleRoleStatus && (
+                <QuickActionConfirmButton
+                    name={undefined}
+                    onConfirm={handleToggleRoleStatus}
+                    title={roleStatus === 'ADMIN' ? 'Unset Admin' : 'Set Admin'}
```
Comment (tnagorra) [inline src/views/Admin/UserRoles/RolesAction/index.tsx:54]:
```suggestion
                    title={roleStatus === 'ADMIN' ? 'Grant Admin Access' : 'Revoke Admin Access'}
```

File: src/views/Admin/UserRoles/RolesAction/index.tsx [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+                    variant={roleStatus === 'ADMIN' ? 'accent' : 'default'}
+                    disabled={disabled || !onToggleRoleStatus}
+                    confirmationMessage={`Are you sure you want to change ${userName}'s role ?`}
```
Comment (tnagorra) [inline src/views/Admin/UserRoles/RolesAction/index.tsx:57]:
```suggestion
                    confirmationMessage={`Are you sure you want to change this user's role ?`}
```

File: src/views/Admin/UserRoles/UserActions/index.tsx [hunk 5]
```diff
@@ -1,12 +1,12 @@
 import React, { useCallback } from 'react';
 import {
-    IoMdCreate,
+    // IoMdCreate,
```
Comment (tnagorra) [inline src/views/Admin/UserRoles/UserActions/index.tsx:3]: Let's just remove these comments

File: src/views/Admin/UserRoles/UserActions/index.tsx [hunk 6]
```diff
@@ -1,12 +1,12 @@
 import React, { useCallback } from 'react';
```
Comment (tnagorra) [inline src/views/Admin/UserRoles/UserActions/index.tsx:1]: Do we even need this component?

File: src/views/Admin/UserRoles/index.tsx [hunk 7]
```diff
@@ -177,12 +216,30 @@ function UserRoles(props: UserRolesProps) {
...
-            const actionColumn: TableColumn<UserRolesField, string, ActionProps, TableHeaderCellProps> = {
+            const actionColumn: TableColumn<
+                UserRolesField,
+                string,
+                ActionProps,
+                TableHeaderCellProps
+            > = {
                 id: 'action',
-                title: 'Actions',
+                title: 'Edit Active Status',
```
Comment (tnagorra) [inline src/views/Admin/UserRoles/index.tsx:242]: We don't need to add any title

Comment (tnagorra) [reply src/views/Admin/UserRoles/index.tsx:242]: We shouldn't need this column anymore

**Review by AdityaKhatri — CHANGES_REQUESTED** [review]

**Review by AdityaKhatri — APPROVED** [review]

**Pull request merged**